eng/common/scripts/Prepare-Release.ps1

```diff
@@ -184,6 +184,13 @@ else
   exit 1
 }
 
+$changelogIsValid = Confirm-ChangeLogEntry -ChangeLogLocation $packageProperties.ChangeLogPath -VersionString $newVersion -ForRelease $true
```

Comment:
Don't we already do this in some of our SetPackageVersion functions? Or did we only update the changelog without validating it?

Reply:
SetPackageVersion only updates the changelog. Here we are validating it like its about to be released, I don't think there is anyother context which we want to do this kind of validation aside from int he pipeline during release.